src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,130 @@
+package duke;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.FindTask;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+import java.util.Scanner;
+
+/**
+ * The Class Duke: driver class.
+ */
+public class Duke {
+
+    /**
+     * The file path.
+     */
+    public String filePath;
+
+    /**
+     * The storage.
+     */
+    private Storage storage;
+
+    /**
+     * The task list.
+     */
+    private TaskList taskList;
+
+    /**
+     * The ui.
+     */
+    private Ui ui;
+
+    /**
+     * Instantiates a new duke.
+     */
+    public Duke() {
+        ui = new Ui();
+        this.filePath = "C:\\Users\\Shaopeng\\Desktop\\duke\\data\\duke.txt";
+        this.storage = new Storage(filePath);
+        try {
+            taskList = new TaskList(storage.loadData());
+        } catch (Exception e) {
+            ui.showLoadingError();
+            taskList = new TaskList();
+        }
+    }
+
+    /**
+     * The main method.
+     *
+     * @param args the arguments
+     */
+    public static void main(String[] args) {
+        new Duke().run();
+    }
+
+    public String getResponse(String input) {
+        return input;
+    }
+
+    /**
+     * This method will be running throughout the entire session
+     * It will listen to the incoming users's command and process the comment accordingly.
+     */
+
+    public void run() {
+        this.ui.greetLogo();
+        this.ui.greet();
+        ui.printInputRequest();
```

Comment:
I think you're missing a "this." in front? Not sure if you did this intentionally